notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1882 @@
+{
```

Comment:
<p>Could you clarify why you're doing this? Is this something we should also document in the public doc?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>As above, it is a feature of KFP. But this is frequently asked by the customers, just put the code here for the convenience.</p>